q2_moshpit/eggnog/_method.py

```diff
@@ -157,3 +158,35 @@ def fetch_eggnog_db() -> EggnogRefDirFmt:
 
     # Return objects
     return eggnog_db
+
+
+def fetch_diamond_db() -> DiamondDatabaseDirFmt:
```

Comment:
For the sake of clarity, could you create a new Python file in this submodule (`dbs.py`) and move all the code relevant to fetching eggnog databases there? Maybe you can also move the exisitng `fetch-eggnog-db` action there.